multiversx_sdk/network_providers/proxy_network_provider.py

```diff
@@ -132,6 +143,41 @@ def send_transactions(self, transactions: Sequence[ITransaction]) -> Tuple[int,
         hashes = response.get("txsHashes")
         return num_sent, hashes
 
+    def await_transaction_completed(self, tx_hash: Union[str, bytes], options: Optional[AwaitingOptions] = None) -> TransactionOnNetwork:
+        if isinstance(tx_hash, bytes):
+            tx_hash = tx_hash.hex()
```

Comment:
As in the other comment, we can have an utility function.

Reply:
added